doc/sources.rst

```diff
@@ -646,3 +648,118 @@ Example:
   debug_source:
     type: debug
 
+
+.. _ogcapitiles_label:
+
+OGC API Tiles
```

Comment:
This section should contain a prominently placed notice/warning that only map tiles and no vector tiles are supported.